wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -77,6 +104,40 @@
   ],
   "emit-alerts": true,
   "emit-errors": true,
+  "filters": [
+    {
+      "name": "PreconditionValueJEXL",
+      "condition": {
+        "expression": "preconditionToggle == 'jexl'"
+      },
+      "show": [
+        {
+          "type": "properties",
+          "name": "directives"
+        },
+        {
+          "type": "properties",
+          "name": "udd"
+        },
+        {
+          "type": "properties",
+          "name": "precondition"
+        }
+      ]
+    },
+    {
+      "name": "preconditionValueSQL",
+      "condition": {
```

Comment:
Similar to above, can we also clear the JEXL precondition field and directives fields when the SQL precondition toggle is selected?